### PR TITLE
Multi category filtering and tab icons

### PR DESCRIPTION
**Pull request by danthecodingman**

### CODE REVIEW

File: fbuapp/View Controllers/CreatePostViewController.m [hunk 1]
```diff
@@ -334,6 +334,9 @@ -(void)didSelectCell: (NSIndexPath *)indexPath {
     self.eventCategory = indexPath.row;
 }
 
+- (void)didDeselectCell: (NSIndexPath *)indexPath{
+}
```
Comment (eazheng) [inline fbuapp/View Controllers/CreatePostViewController.m:338]: should there be stuff in this function

File: fbuapp/View Controllers/FilterViewController.m [hunk 2]
```diff
@@ -155,7 +164,19 @@ -(BOOL)isValidNumber: (NSString *) string{
...
+
+- (void)didDeselectCell: (NSIndexPath *)indexPath{
+    [self.eventCategory removeObject:@(indexPath.row)];
+    if([self.eventCategory count] > 0)
+    {
+        [self.postQuery whereKey: @"eventCategory" containedIn:self.eventCategory];
+        
+    }
+    else{
+        [self.postQuery whereKey: @"eventCategory" containedIn:@[@0, @1, @2, @3, @4, @5, @6, @7]];
```
Comment (eazheng) [inline fbuapp/View Controllers/FilterViewController.m:178]: is there a way to clear the query for "eventCategory" without having to list out the entire array 0 to 7

File: fbuapp/View Controllers/HomeViewController.m [hunk 3]
```diff
@@ -156,8 +161,22 @@ - (void) showDetails: (Post *)post {
...
+
+- (void)didDeselectCell: (NSIndexPath *)indexPath{
+    [self.savedQuery.category removeObject:@(indexPath.row)];
+    if([self.savedQuery.category count] > 0)
+    {
+        [self.postQuery whereKey: @"eventCategory" containedIn:self.savedQuery.category];
+
+    }
+    else{
+        [self.postQuery whereKey: @"eventCategory" containedIn:@[@0, @1, @2, @3, @4, @5, @6, @7]];
```
Comment (eazheng) [inline fbuapp/View Controllers/HomeViewController.m:178]: same here

File: fbuapp/View Controllers/CreatePostViewController.m (hunk 1 above)
Comment (danthecodingman) [reply fbuapp/View Controllers/CreatePostViewController.m:338]: no but this method is necessary for the delegate

File: fbuapp/View Controllers/FilterViewController.m (hunk 2 above)
Comment (danthecodingman) [reply fbuapp/View Controllers/FilterViewController.m:178]: In order to override the previous whereKey containedIn query on postQuery we must use the same query method, so this is the only way

File: fbuapp/View Controllers/FilterViewController.m [hunk 4]
```diff
@@ -155,7 +164,19 @@ -(BOOL)isValidNumber: (NSString *) string{
...
+    }
 }
 /*
 #pragma mark - Navigation
```
Comment (belchercd) [inline fbuapp/View Controllers/FilterViewController.m:182]: delete

**Pull request merged**